README.md

```diff
@@ -74,6 +105,58 @@ Severity levels could be: `error`, `warning`, `ok`
 
 Please notice that Grafanalabs uses its own configuration for plugins submissions and your own config file can't change these rules.
 
+## Debug mode
+
+You can run the validator in debug mode to get more information about the running checks and possible errors.
+
+With Docker:
+
+```SHELL
+docker run --pull=always -e DEBUG=1 grafana/plugin-validator-cli -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
+```
+
+With NPX:
+
+```SHELL
+DEBUG=1 npx -y @grafana/plugin-validator -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
+```
+
+Locally:
+
+```SHELL
+DEBUG=1 plugincheck2 -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
+```
+
+## Sourcecode and Git repositories
+
+You may pass the sourceCodeUri to the validator in order to perform source code checks (`-sourceCodeUri` option).
+
+### Using git URLs
+
+The following git services are supported
+
+- Github
```

Comment:
minor: welcome to dismiss, but should be GitHub and GitLab

Reply:
I'm putting github first as it is the most popular service (so it comes to mind first)

Reply:
just to be clear - my point is on the capitalization rather than ordering etc

Reply:
oh! I see. I'll fix it.